torch/csrc/jit/passes/onnx.cpp

```diff
@@ -331,6 +332,7 @@ void BlockToONNX(
     // Call the symbolic function
     // Use a little trampoline function so we can give good error messages
     // upon argument mismatch
+    onnx_registry.attr("register_op")(op->name(), pyobj.attr("symbolic"), "", 0);
```

Comment:
Why pass 0 as the opset_version here? As we discussed in the proposal, we should pass in the target opset, if the target symbolic is missing, we should fall back to the older/previous opset.